src/stylesheets/components/_accordions.scss

```diff
@@ -1,16 +1,12 @@
 // Variables
 
-$accordion-border: 3px solid $color-gray-lightest;
+$accordion-border: 3px solid color('gray-5');
```

Comment:
- Let's use the theme palette var here: `color('base-lighter')`
- Let's look at using `2px` instead of `3px` to align with system palette